src/api/app/views/webui/comment/_reply.html.haml

```diff
@@ -15,6 +15,11 @@
       = link_to('#', data: { 'bs-toggle': 'modal', 'bs-target': "#delete-comment-modal-#{comment.id}" },
                 class: 'delete_link btn btn-outline-danger', title: 'Delete comment') do
         Delete
+  - if Flipper.enabled?(:content_moderation, User.session)
+    - if policy(comment).moderate?
```

Comment:
What about moving the flipper check inside the policy?

Reply:
I finally left it as it was.

Reply:
Alright